[2024-03-04T17:42:38.905+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.group_database.create_table_task manual__2024-03-04T17:31:20.572719+00:00 [queued]>
[2024-03-04T17:42:38.912+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.group_database.create_table_task manual__2024-03-04T17:31:20.572719+00:00 [queued]>
[2024-03-04T17:42:38.912+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T17:42:38.912+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-03-04T17:42:38.912+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T17:42:38.920+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): group_database.create_table_task> on 2024-03-04 17:31:20.572719+00:00
[2024-03-04T17:42:38.924+0000] {standard_task_runner.py:55} INFO - Started process 150 to run task
[2024-03-04T17:42:38.926+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gera_dados_turbina', 'group_database.create_table_task', 'manual__2024-03-04T17:31:20.572719+00:00', '--job-id', '3362', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpr0b431gs']
[2024-03-04T17:42:38.928+0000] {standard_task_runner.py:83} INFO - Job 3362: Subtask group_database.create_table_task
[2024-03-04T17:42:38.966+0000] {task_command.py:388} INFO - Running <TaskInstance: gera_dados_turbina.group_database.create_table_task manual__2024-03-04T17:31:20.572719+00:00 [running]> on host 06e5b3fc014c
[2024-03-04T17:42:39.009+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=adrianolima095@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gera_dados_turbina
AIRFLOW_CTX_TASK_ID=group_database.create_table_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-04T17:31:20.572719+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-04T17:31:20.572719+00:00
[2024-03-04T17:42:39.009+0000] {sql.py:254} INFO - Executing: create table if not exists sensors 
                                        (  idtemp            varchar
                                         , powerfactor       varchar
                                         , hydraulicpressure varchar
                                         , temperature       varchar
                                         , timestamp         varchar
                                         );
                                    
[2024-03-04T17:42:39.014+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2024-03-04T17:42:39.157+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2024-03-04T17:42:39.160+0000] {sql.py:375} INFO - Running statement: create table if not exists sensors 
                                        (  idtemp            varchar
                                         , powerfactor       varchar
                                         , hydraulicpressure varchar
                                         , temperature       varchar
                                         , timestamp         varchar
                                         );
                                    , parameters: None
[2024-03-04T17:42:39.166+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=gera_dados_turbina, task_id=group_database.create_table_task, execution_date=20240304T173120, start_date=20240304T174238, end_date=20240304T174239
[2024-03-04T17:42:39.219+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-03-04T17:42:39.229+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
